src/middlewared/middlewared/validators.py

```diff
@@ -209,6 +210,58 @@ def __call__(self, value):
             raise ValueError(f'Invalid UUID: {e}')
 
 
+class PasswordComplexity:
+    def __init__(self, required_types, required_cnt=None):
+        self.required_types = required_types
```

Comment:
If an invalid type is specified, what happens ?

Reply:
Hmm... Linux doesn't have EDOOFUS 🤔